testsuite/features/support/custom_formatter.rb

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) 2019 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+### This file is not indicated as covered however the coverage might be implicit?
+
```

Comment:
Not sure if the code is actually unused or if the tool failed to consider it.

Reply:
We are using it from outside, from the Rakefile.

```
      cucumber_opts = "#{include_profiles} #{include_tags} -f html -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.html -f json -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.json #{junit_results} -f CustomFormatter::PrettyFormatter -r features "
```